package.json

```diff
@@ -232,9 +239,10 @@
     "cross-env": "^7.0.2",
     "css-loader": "^5.2.6",
     "detect-port": "^1.3.0",
-    "electron": "^13.1.6",
-    "electron-builder": "^22.11.7",
-    "electron-devtools-installer": "^3.2.0",
+    "electron": "^12.0.2",
+    "electron-builder": "^22.9.1",
+    "electron-devtools-installer": "^3.1.1",
+    "electron-notarize": "^1.0.0",
```

Comment:
Going back was needed?

Reply:
Ah, inadvertent change, thanks for picking that up :)